sbt.yml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  annotations:
+    description: Build and run Scala and Java applications using SBT
+    iconClass: icon-scala
+    openshift.io/display-name: SBT
+    openshift.io/image.dockerRepositoryCheck: 2017-12-08T19:01:11Z
+    eationTimestamp: 2017-12-08T18:32:08Z
```

Comment:
Looks like a typo `eationTimestamp`.

State metadata should be cut out of this template.  

So at least this and the `dockerRepositoryCheck` above can go.